src/sidebar/templates/group-section.html

```diff
@@ -0,0 +1,13 @@
+<h2 class="heading" ng-if="vm.sectionGroups.length > 0">
+  {{ vm.heading }}
+</h2>
+<ul class="content" ng-if="vm.sectionGroups.length > 0">
+  <li class="dropdown-community-groups-menu__row dropdown-menu__row--unpadded"
+      ng-repeat="group in vm.sectionGroups track by group.id">
+
+    <group class="group" group="group" ng-if="vm.isSelectable(group.id)"></group>
```

Comment:
In Angular, an actual DOM element gets created for custom components like `<group>`, so the `class` property will have an effect. In React however this is not the case. The component tree is separate from the "rendered tree" of DOM elements. Therefore it will make the migration easier if classes are only applied to DOM elements (the top-level element inside `group.html` in this case) rather than to components. The exception is if for some reason you need to style different uses of the component with different classes.

Reply:
Yeah I didn't want to have to style it but it was causing a problem (shrinking the group row). It's styling is the same as the inner component so I expect when we move to React that styling will no longer be needed. If you can see a way to avoid it lmk (my css foo is somewhat limited).